doc/PLUGINS.md

```diff
@@ -223,6 +223,19 @@ to a peer was lost.
 }
 ```
 
+#### `invoice_payment`
+
+A notification for topic `invoice_payment` is sent every time an invoie is paid.
+
+```json
+{
```

Comment:
Is this not wrapped in an `invoice_payment` object?  From my reading of the code it should be similar to the `warning` notification which has a `warning` object wrapping it.

Reply:
I forgot to updated the docs, thanks !